### DISCUSSION TITLE
How to create edge label like in demo

### DISCUSSION THREAD

**Discussion by ghost** [xyflow/xyflow#3004] [Q&A] [answered]
![Screenshot_20230419_151708_Chrome](https://user-images.githubusercontent.com/6542274/233037924-8d267819-d871-4de8-981c-be85c853debf.jpg)
In this screenshot animated text with an arrow points to a node. How to create it?

**Comment by bcakmakoglu** [comment]
Those aren't actually edge labels but custom nodes.
[The example can be found here.](https://github.com/wbkd/react-flow-docs/blob/main/src/components/HeroFlow/index.js)

**Reply by ghost** [reply]
These look nice. It would be good to provide them a meaningful term than custom node. For example, edge, node and what would be it's name?

**Reply by bcakmakoglu** [reply]
Swoopy nodes is what they’re called 😄

**Reply by bcakmakoglu** [reply, marked as answer]
[FYI here's the actual code for the swoopy nodes.](https://github.com/wbkd/react-flow-docs/blob/main/src/components/HeroFlow/SwoopyNode.js)
